tests/test_response.py

```diff
@@ -85,6 +86,44 @@ def test_response_body_passes_through_when_base64(self) -> None:
             (666, False),
         ],
     )
-    def test_response_is_ok(self, code: int, outcome: bool) -> None:
+    def test_response_ok(self, code: int, outcome: bool) -> None:
         response = Response({"message": "xxx"}, headers={"xx": "xx"}, status_code=code)
+
+        assert response.ok == outcome
         assert response.is_ok() == outcome
+
+    def test_from_exception(self) -> None:
+        response = Response.from_exception(ServerError(), "req-id")
+
+        assert response.to_dict() == {
+            "body": '{"message":"Server got itself in trouble","request_id":"req-id"}',
+            "headers": {"Content-Type": "application/json"},
+            "isBase64Encoded": False,
+            "statusCode": 500,
+        }
+
+    def test_from_exception_with_error_code(self) -> None:
+        class RandomException(LambdaFWException):
+            error_code = "RAND001"
+
+        response = Response.from_exception(RandomException(), "req-id")
+
+        assert response.to_dict() == {
+            "body": '{"message":"Server got itself in '
+            'trouble","request_id":"req-id","error_code":"RAND001"}',
+            "headers": {"Content-Type": "application/json"},
+            "isBase64Encoded": False,
+            "statusCode": 500,
+        }
+
+    def test_json_when_body_is_a_string(self) -> None:
+        response = Response('{"message":"Server got itself in trouble"}')
+
+        assert response.json() == {
+            "message": "Server got itself in trouble",
+        }
+
+    def test_json_when_body_is_a_dict(self) -> None:
+        response = Response({"some": "data"})
+
+        assert response.json() == {"some": "data"}
```

Comment:
It looks like it can be simply parameterized:
```suggestion
    @pytest.mark.parametrize(
        "body",
        [
            {"message": "It is alive!"},
            '{"message":"It is alive!"}',
        ]
    )
    def test__json__returns_dict_based_on_declared_body(self, body: Any) -> None:
        response = Response(body)

        assert response.json() == {"message": "It is alive!"}
```